Source/VCGeneration/ProverInterface.cs

```diff
@@ -156,12 +158,7 @@ public virtual Absy Label2Absy(string label)
   public abstract void BeginCheck(string descriptiveName, VCExpr vc, ErrorHandler handler);
 
   [NoDefaultContract]
-  public abstract Outcome CheckOutcome(ErrorHandler handler, int errorLimit);
-
-  public virtual string[] CalculatePath(int controlFlowConstant)
-  {
-    throw new System.NotImplementedException();
-  }
```

Comment:
Oh, nice! This doesn't need to be here?

Reply:
Not for `https://github.com/boogie-org/boogie`, but possibly for a consumer of `https://github.com/boogie-org/boogie`, although I don't see why you would want to override this. For APIs that are [unexpectedly] used outside of Boogie, we should document the use-case and mark the API with an annotation, like `[PublicAPI]`, otherwise it'll be impossible to make changes to Boogie without fear of breaking a consumer.